packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactEditText.java

```diff
@@ -269,6 +303,36 @@ public InputConnection onCreateInputConnection(EditorInfo outAttrs) {
     return inputConnection;
   }
 
+  /*
+   * Called when a context menu option for the text view is selected.
+   * React Native replaces copy (as rich text) with copy as plain text.
+   */
+  @Override
+  public boolean onTextContextMenuItem(int id) {
+    if (id == android.R.id.paste) {
+      if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
+        id = android.R.id.pasteAsPlainText;
+      } else {
+        ClipboardManager clipboard =
+            (ClipboardManager) getContext().getSystemService(Context.CLIPBOARD_SERVICE);
+        ClipData clip = clipboard.getPrimaryClip();
+        if (clip != null) {
+          for (int i = 0; i < clip.getItemCount(); i++) {
+            final CharSequence text = clip.getItemAt(i).coerceToText(getContext());
+            final CharSequence paste = (text instanceof Spanned) ? text.toString() : text;
+            if (paste != null) {
+              ClipData clipData = ClipData.newPlainText("rebase_copy", text);
```

Comment:
nit: why this label?

Reply:
Thanks @NickGerleman I changed the label to null as implemented in TextView#onTextContextMenuItem for ID_COPY.

https://github.com/aosp-mirror/platform_frameworks_base/blob/27c5bed4fa582fa934da55528c12af414167de8d/core/java/android/widget/TextView.java#L12928

Relevant https://github.com/aosp-mirror/platform_frameworks_base/blob/27c5bed4fa582fa934da55528c12af414167de8d/core/java/android/content/ClipData.java#L849